app/database/accessor/breeze_buddy/outbound_number_pool.py

```
@@ -0,0 +1,240 @@
+async def increment_pool_channels(
+    pool_id: str,
+) -> Optional[OutboundNumberPool]:
+    """Atomically increment current_channels. Returns None if at capacity."""
+    try:
+        query, values = increment_pool_channels_query(pool_id)
+        result = await run_parameterized_query(query, values)
+        if get_row_count(result) == 0:
+            logger.warning(f"Pool {pool_id} at capacity — increment returned zero rows")
+            return None
+        return decode_outbound_number_pool(result)
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)